gatherers/manifest.js

```diff
@@ -61,32 +71,29 @@ class Manifest extends Gather {
      * potentially lead to a different asset. Using the original manifest
      * resource is tracked in issue #83
      */
-    return driver.querySelector('head link[rel="manifest"]')
-      .then(node => {
-        if (!node) {
-          this.artifact = Manifest._errorManifest('No <link rel="manifest"> found in DOM.');
-          return;
-        }
+    return driver.sendCommand('Runtime.evaluate', {
+      expression: getManifestContent,
+      returnByValue: true
+    }).then(returnedData => {
+      if (returnedData.result.value === undefined ||
+          returnedData.result.value === null ||
+          returnedData.result.value === {}) {
+       // The returned object from Runtime.evaluate is an enigma
+       // Sometimes if the returned object is not easily serializable,
+       // it sets value = {}
+        throw new Error('Failed to get proper result from runtime eval');
```

Comment:
True. There is no way to indicate a particular gather failed, is there? I also really wanted a failed gather to not stop the other gatherers since they're all independent now. I can just include the name of the gatherer here for now.